lib/assessment/code_feature_extractor.py

```diff
@@ -283,21 +279,24 @@ def extract_movement_types(self, node):
             if obj:
               if isinstance(statement["value"], dict):
                 if "left" in statement["value"]:
-                  if statement["assignee"] in [statement["value"]["left"], statement["value"]["right"]]:
-                    if statement["value"]["operator"] in ["+", "-"]:
-                      self.features["movement"]["counter"] += 1
-                      self.nodes.append(node)
+                  if statement["assignee"] in [statement["value"]["left"], statement["value"]["right"]] and statement["value"]["operator"] in ["+", "-"]:
+                    self.features["movement"]["counter"]["count"] += 1
+                    self.features["movement"]["counter"]["lines"].append({'start': statement["start"], 'end': statement["end"]})
+                    self.nodes.append(node)
```

Comment:
I don't think this needs to happen in this PR, but another refactor I've been thinking about a bit is merging the movement counter function with the property change extractor. I think there's a lot of overlap already since movement is a type of property change (changing the x or y property), and the property change extractor also labels if a feature is in the draw loop or not. It's not a fully baked idea yet, but we can look at it more together later